pennylane/devices/default_qubit_jax.py

```diff
@@ -229,3 +230,59 @@ def states_to_binary(samples, num_wires, dtype=jnp.int32):
         powers_of_two = 1 << jnp.arange(num_wires, dtype=dtype)
         states_sampled_base_ten = samples[:, None] & powers_of_two
         return (states_sampled_base_ten > 0).astype(dtype)[:, ::-1]
+
+    def estimate_probability(self, wires=None, shot_range=None, bin_size=None):
+        """Return the estimated probability of each computational basis state
```

Comment:
If this docstring is the same as the one in `QubitDevice`, then maybe we can get away with redefining it (due to inheritance).